src/store/ducks/plans.ts

```diff
@@ -13,11 +13,13 @@ export enum InterventionType {
   IRS = 'IRS',
 }
 
+/** interface for plan Objects */
 /** Enum representing the possible intervention types */
 export enum PlanStatus {
   ACTIVE = 'active',
   COMPLETE = 'complete',
   DRAFT = 'draft',
+  NEW = 'new',
```

Comment:
I think `new` is not a supported plan status value from the FHIR docs on plan definitions.  (`complete` is also not supported by FHIR but we chose to add it in).  I was talking to @rowo in another issue on this repo and we think we can use the plan date field to kind of loosely determine whether a plan is new or not.

@githengi could you please confirm if I got this right?

Reply:
I think `new` is not required. New plans should be of status `draft` or what kind of action is needed to move a plan from `new` to `draft`?

Reply:
`new` was meant to be more of an internal status representing a plan that has not yet been saved as a `draft`, the only action required is for the user to click the `save as draft` button and the draft plan model is constructed before being saved to OpenSRP with `status: draft`.

Reply:
aaaah, cool.  It makes it a little complicated to deal with this enum coz you ave to keep in mind one of them should not be used/expected with the OpenSRP server and/or Superset

Reply:
> you ave to keep in mind one of them should not be used/expected

Good point, I'll pull it out of the type, add the string "new" to a constant, and set the component state to allow either the enum or the constant.

Reply:
@moshthepitt - I tried quickly decoupling `new` out of the `PlanStatus` but started running into some gnarly TS errors about extending / inheritance. I think this should be done but maybe for now we can just wrap the service calls with conditionals to protect making calls to OpenSRP with `new`.